### PR TITLE
Remove command

### PR DESCRIPTION
**Pull request by jburel**
Use nio package to create hard link and symlink

Test the various in-place import options on several OS.

cc @joshmoore @mtbc @pwalczysko

### CODE REVIEW

**Comment by joshmoore** [comment]
The error code handling from `exec` will need to be resurrected as exception handling. Specifically the error messages were requested so that users could figure out what was going on.

Also, since `createProcessBuilder` is protected, this would count as a breaking change since it would no longer do anything for subclasses which implement the `createProcessBuilder` method. Options I can think of:

 * keep `createProcessBuilder` but return an implementation which calls the native Java method
 * leave AbstractExecFileTransfer` as it is, and implement add new classes (`Hardlink2FileTransfer extends AbstractFileTransfer` (that may require moving some of the methods from AEFT up to AFT)

**Comment by jburel** [comment]
@joshmoore: thanks I was not sure about the plan for error handling since the method already returns IOException (not handled)

**Comment by jburel** [comment]
I have introduce new classes. I opted for a new ```AbstractExecFileTransfer2```.
The simplest thing might be to mark some classes deprecated this round and use 5.3 for an update.
1.6 support was dropped a while back and those classes never got cleaned up.

File: components/blitz/src/ome/formats/importer/transfers/AbstractExecFileTransfer.java [hunk 1]
```diff
@@ -74,83 +68,6 @@ public String transfer(TransferState state) throws IOException, ServerError {
     }
 
     /**
-     * Build a path of the form "root.path/root.name/file.path/file.name".
-     *
-     * @param root the root directory
-     * @param ofile a path relative to the root
-     * @return the assembled path with separators suitable for the local filesystem
-     */
-    protected File getLocalLocation(OriginalFile root, OriginalFile ofile) {
```
Comment (joshmoore) [inline components/blitz/src/ome/formats/importer/transfers/AbstractExecFileTransfer.java]: I'd still avoid the removal of these `protected` methods.

Comment (joshmoore) [inline components/blitz/src/ome/formats/importer/transfers/AbstractExecFileTransfer.java]: Sorry, see them below.

**Comment by joshmoore** [comment]
Excluding for the moment: https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-integration-python/lastCompletedBuild/testReport/OmeroPy.test.integration.clitest.test_import/TestImport/testSymlinkImport/ et al.

**Comment by jburel** [comment]
Sorry I did not have a chance to run the tests. I will try to do that this pm

**Comment by jburel** [comment]
Another example of testing headache (plenty with Ice 3.6)
run locally
```test/integration/clitest/test_import.py::TestImport::testSymlinkImport PASSED```

**Comment by jburel** [comment]
@joshmoore @sbesson:
run the tests on breaking with this PR included
no problem see https://ci.openmicroscopy.org/view/OMERO-DEV/job/OMERO-DEV-breaking-integration-python/491/

**Comment by sbesson** [comment]
@jburel: interestingly this PR was not picked in the merge job - see https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-breaking-push/623/.

**Comment by jburel** [comment]
hummm

**Comment by jburel** [comment]
Crap It did not pick when I remove the exclude. 
I am going to run it again

**Comment by jburel** [comment]
Few errors on breaking. I will investigate when I get back since the Symlink test passes locally

**Comment by jburel** [comment]
Tested locally symlink creation with both Java 1.7 and 1.8 no problem noticed

**Comment by jburel** [comment]
The difference between local and server is the java version installed. I will run some tests on docker now that we have the various parameters in place.
Closing this PR for now.

**Pull request closed without merging**